src/transactions/lib.js

```diff
@@ -1,6 +1,19 @@
 import { ids } from "./ids";
 import { getAppByID } from "../wallets/wallets";
 
+export let cdpInterest = .02; // XXX: This should be kept close to the actual interest rate - it is updated on initialization though
+
+export async function getInterest() {
+  // TODO: cache interest
+  console.log("getInterest called")
+  const interestInfo = await getAppField(ids.app.dao.interest, "interest_rate")
+  console.log("getInterest returned: ", interestInfo)
+  return interestInfo / 1000
+}
+
+// We immeadiately update the interest in a background thread
+getInterest()
```

Comment:
This doesn't update the exported interest rate. It's return value isn't captured so why is it even called?

Reply:
Fixed